Examen/CT/src/exercice2_2/MaskedRaster.java

```diff
@@ -0,0 +1,35 @@
+package exercice2_2;
+
+public class MaskedRaster {
+	public int[][] bits;
+	public boolean[][] executionMask;
+		
+	public MaskedRaster(int[][] input) {
+		super();
+		// TODO Auto-generated constructor stub
+		executionMask = new boolean[input.length][input[0].length];
+		
+		for (int i = 0; i < executionMask.length;i++) {
+            for (int j =0; j < executionMask[0].length;j++) {
+            	executionMask[i][j]= false;
+            }
+        }
+	}
+	
+	private void initMask() {
+		int[][] result = new int [bits.length][bits[0].length];
```

Comment:
WTF??!!!
Come on!!!! Read the test carefully!!!!
You've implemented in `initMask` what was expected in `getBits`